shared/actions/tracker.js

```diff
@@ -11,7 +11,7 @@ import openUrl from '../util/open-url'
 import {requestIdleCallback} from '../util/idle-callback'
 import {isMobile} from '../constants/platform'
 import {type TypedState} from '../constants/reducer'
-import type {FriendshipUserInfo} from '../profile/friendships'
```

Comment:
moved into types. Profile code is VERY old and creaky. Didn't want to change it too much so just moving this out of a render file into types